programs/rewards/src/asserts.rs

```diff
@@ -88,6 +72,19 @@ pub fn assert_pubkey_eq(given: &Pubkey, expected: &Pubkey) -> ProgramResult {
     }
 }
 
+pub fn assert_account_len(account: &AccountInfo, len: usize) -> ProgramResult {
+    if account.data_len() >= len {
```

Comment:
shouldn't we check for just equality here?

Reply:
Why would it harm if an account is bigger for some reason?

Reply:
The only potential problem is if it's a different (bigger) account with the same owner, so we could spoil or misinterpret some data.

Reply:
Hmm, but we read the precise number of bytes. Even if the passed account is 9999k long, we will read only `LEN`  bytes. It's not hard to fix, just wondering.